src/Builtins.jl

```diff
@@ -19,24 +13,36 @@ end
 `@bind x Slider(1:10; default=8, show_value=true)`
 
 """
-Slider(range::AbstractRange; default=missing, show_value=false) = Slider(range, (default === missing) ? first(range) : default, show_value)
-
-function show(io::IO, ::MIME"text/html", slider::Slider)
-    print(io, """<input 
-        type="range" 
-        min="$(first(slider.range))" 
-        step="$(step(slider.range))" 
-        max="$(last(slider.range))" 
-        value="$(slider.default)"
-        $(slider.show_value ? "oninput=\"this.nextElementSibling.value=this.value\"" : "")
-        >""")
-    
-    if slider.show_value
-        print(io, """<output>$(slider.default)</output>""")
-    end
-end
-
-get(slider::Slider) = slider.default
+Slider(range::AbstractRange; default=first(range), show_value=false) = @htl("""
+<div>
+	<input type="range" min="$(first(range))" max="$(last(range))" step="$(step(range))" value="$(default)">
+	$(HTML(show_value ? "<output>$(default)</output>" : ""))
+	
+	<script>
+		let div = currentScript.parentElement
+		let slider = div.querySelector("input")
+	
+		slider.addEventListener("input", e => {
+			div.value = +slider.value
+			$(JavaScript(show_value ? "div.querySelector(\"output\").value = +slider.value" : ""))
+			div.dispatchEvent(new CustomEvent("input"))
+			e.preventDefault()
+		})
+	
+		div.value = $(default)
+		let localVal = div.value
+		delete div.value
```

Comment:
What is this?

Reply:
The main issue here is that the reason that previously sliders used to work (i.e. the sliders will sync across bonds), with multiple of them, is that with the new dral-base synchronization the first, outermost tag will update the value property (N.B. not attribute)

For input ranges, this means that we can see them move back and forth. The code here is a trick, by wrapping it in a div, and overwriting the value setter, we can synchronize the update across both the input slider and the output number that occurs with `show_value=true`

Reply:
Sorry for the vague comment, I meant: why do you `delete div.value`? What happens if you don't?

Reply:
Ah. It makes sense in my head that to create a new property, one should remove the previous one. But you might be right that it makes no difference. I don't know enough javascript to comment on whether it does.

Ill play around tomorrow and see if not having the `delete` makes any visible changes to the Slider. If not, I can remove it from the code.

Reply:
Indeed it doesn't seem to make a difference. From my readings, it shouldn't make a difference as I immediately redefine the property.